StarkInfra/StarkInfra/IssuingEmbossingRequest/Log.cs

```diff
@@ -0,0 +1,196 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using StarkInfra.Utils;
+
+
+namespace StarkInfra
+{
+    public partial class IssuingEmbossingRequest
+    {
+        /// <summary>
+        /// IssuingEmbossingRequest.Log object
+        /// <br/>
+        /// Every time an IssuingEmbossingRequest entity is modified, a corresponding IssuingEmbossingRequest.Log
+        /// is generated for the entity. This log is never generated by the
+        /// user, but it can be retrieved to check additional information
+        /// on the IssuingEmbossingRequest.
+        /// <br/>
+        /// Properties:
+        /// <list>
+        ///     <item>ID [string]: unique id returned when the log is created. ex: "5656565656565656"</item>
+        ///     <item>Request [IssuingEmbossingRequest]: IssuingEmbossingRequest entity to which the log refers to.</item>
+        ///     <item>Type [string]: type of the IssuingEmbossingRequest event which triggered the log creation. ex: "blocked", "canceled", "created", "expired", "unblocked", "updated"</item>
+        ///     <item>Errors [list of strings]: list of errors linked to this IssuingEmbossingRequest event</item>
+        ///     <item>Created [DateTime]: creation datetime for the log. ex: new DateTime(2020, 3, 10, 10, 30, 0, 0)</item>
+        /// </list>
+        /// </summary>
+        public class Log : Resource
+        {
+            public DateTime Created { get; }
+            public string Type { get; }
+            public List<Dictionary<string, object>> Errors { get; }
```

Comment:
se n for muito trampo tenta deixar List<StarkInfra.Error>

Reply:
Mano, todos os recursos estão utilizando o `List<Dictionary<string, object>> Errors` não é melhor seguir o que os outros recursos estão fazendo e posteriormente mudamos pra todo os recursos para List<StarkInfra.Error>?